go.mod

```diff
@@ -35,7 +36,10 @@ require (
 	golang.org/x/sync v0.7.0
 	golang.org/x/term v0.22.0
 	gopkg.in/yaml.v3 v3.0.1
+	gorm.io/gorm v1.25.12
 	gotest.tools/v3 v3.5.1
+	k8s.io/apimachinery v0.31.1
+	k8s.io/apiserver v0.31.1
```

Comment:
I got approval from Darren to include whatever k8s dependencies were necessary to do encryption the same way we did it in mink.